opencog/openpsi/utilities.scm

```diff
@@ -326,13 +326,13 @@
 		result
 		; else check if entity is an evaluation or predicate or schema
 		(let ((type (cog-type entity)))
-			(if (or (equal? type 'PredicateNode)
-				   (equal? type 'GroundedPredicateNode)
+			(if (or (equal? type 'GroundedPredicateNode)
 				   (equal? type 'DefinedPredicateNode))
 				(set! result (cog-evaluate! (Evaluation entity (List)))))
-			(if (or (equal? type 'SchemaNode)
-            		(equal? type 'GroundedSchemaNode)
-            	    (equal? type 'DefinedSchemaeNode))
+			(if (equal? type 'PredicateNode)
+				(set! result (cog-tv entity)))
```

Comment:
@bgoertzel What about the issue of (a) using EvaluationLink for assigning values to predicates with no arguments (for consistency with predicates that do have arguments) vs. (b) using the PredicateNode TV (rather than EvaluationLink) to assign value to predicates that don't have arguments?

It seems to me there should be an OpenCog standard here. For example, if openpsi wants to know the value of a 0-ary predicate that is defined by a different component of the system (e.g., a PAU like voice-width), where should openpsi look to get it's value from, the PredicateNode TV or a wrapped EvaluationLink TV?